packages/fx-core/src/plugins/solution/fx-solution/arm.ts

```diff
@@ -399,10 +401,15 @@ export async function deployArmTemplates(ctx: SolutionContext): Promise<Result<v
         [SolutionTelemetryProperty.Success]: SolutionTelemetrySuccess.Yes,
       });
     } else {
+      const properties: { [key: string]: string } = {};
+      if (result.error.innerError) {
+        properties[SolutionTelemetryProperty.ModuleErrorMessage] = result.error.innerError;
```

Comment:
can we name the property to `arm-deployment-error` or something else to make it more straightforward?